UDLs/STL-3dObject-ASCII.byPryrt.xml

```diff
@@ -0,0 +1,64 @@
+<NotepadPlus>

```

Comment:
Sorry, my submission of this file technically should have included a description of the language to prove general-interest: https://en.wikipedia.org/wiki/STL_(file_format).